pom.xml

```diff
@@ -61,6 +71,14 @@
                     <webXml>src/main/webapp/web.xml</webXml>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
does it work without this plugin? you already have it in `<pluginManagement>` section

Reply:
done